wagtail/admin/views/generic/models.py

```diff
@@ -256,13 +256,29 @@ def get_buttons(column, instance, *args, **kwargs):
         )
 
     def _get_custom_column(self, field_name, column_class=Column, **kwargs):
-        label, attr = label_for_field(field_name, self.model, return_attr=True)
+        *relation, field = field_name.split(".")
+        model_class = self.model
+
+        # iterate over the relation list to try to get the last model
+        # where the field exists
+        for model in relation:
+            model_class = getattr(model_class, "_meta").get_field(model).related_model
```

Comment:
Is there a reason why we use `getattr()` here instead of `model_class._meta` when we're always accessing `_meta` and we're not using a default fallback value?

Reply:
Tbh, I don't remember why I used `getattr()` instead of directly accessing it using `model_class._meta`. I changed it to `model_class._meta`.